e2e/app/component/table.ts

```diff
@@ -90,11 +99,50 @@ export default class Table extends Container {
     return this.page.waitForXPath(cellXpath, { visible: true });
   }
 
+  async getCellByValue(rowValue: string, columnHeader: string): Promise<Cell | null> {
+    // Find the row which contains cell
+    let rowIndex = -1;
+    const rows = await this.getRows();
+    for (let i = 0; i < rows.length; i++) {
+      const tds = await rows[i].$x('./td');
+      for (let j = 0; j < tds.length; j++) {
+        const v = await getPropValue<string>(tds[j], 'innerText');
+        if (v === rowValue) {
+          rowIndex = i;
+          break;
+        }
+      }
+    }
```

Comment:
I could use some help to rewrite this block of code to functional code.

Reply:
OK - this would be a fun mob activity.  Let's merge the PR first, though